doc/conf.py

```diff
@@ -61,7 +61,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'alabaster'
+html_theme = 'sphinx_rtd_theme'
```

Comment:
Is this required?

Reply:
No, but doesn't it look nicer? sphinx_rtd_theme is the "standard" RTD theme, alabaster is here: https://alabaster.readthedocs.io/en/latest/

Reply:
I kind of like alabaster better. :)

Reply:
I've added that to the agenda for Friday. :P

Reply:
> I've added that to the agenda for Friday. :P

:cold sweat:

I'll set it back to alabaster 😃